tests/e2e/support/devices.js

```diff
@@ -0,0 +1,19 @@
+const devices = [
```

Comment:
Just curious, why we have a different set of devices for 3key-generator tests and the explorer ones?